evalml/tests/conftest.py

```diff
@@ -2529,6 +2529,75 @@ def _return_proper_func(real_or_synthetic):
     return _return_proper_func
 
 
+@pytest.fixture
+def generate_multiseries_seasonal_data():
+    """Function that returns data with a linear trend and a seasonal signal with specified period for multiseries."""
+
+    def generate_synthetic_data(
+        period,
+        step=None,
+        num_periods=20,
+        scale=1,
+        seasonal_scale=1,
+        trend_degree=1,
+        freq_str="D",
+        set_time_index=False,
+    ):
+        """Function to generate a sinusoidal signal with a polynomial trend.
+
+        Args:
+            period: The length, in units, of the seasonal signal.
+            step:
```

Comment:
Is `step` needed for this function? If so then make sure to fill out what it does here.

Reply:
Yeah looks like it was just left over from the `generate_seasonal_data` function that I copied but I'll take it out here